libkbfs/keybase_service_base.go

```diff
@@ -355,7 +361,8 @@ func (k *KeybaseServiceBase) StartReachability(ctx context.Context) (res keybase
 // CheckReachability implements keybase1.ReachabilityInterface.
 func (k *KeybaseServiceBase) CheckReachability(ctx context.Context) (res keybase1.Reachability, err error) {
 	res.Reachable = keybase1.Reachable_NO
-	if k.config.MDServer().IsConnected() {
+	mdServer := k.config.MDServer()
+	if mdServer != nil && mdServer.IsConnected() {
```

Comment:
If `mdServer` is `nil`, might wanna wait for a few seconds and retry here? Otherwise caller might try to reconnect too soon when we are still starting up.

Reply:
The retries should happen upstream, shouldn't they?

Reply:
Yes, that's the reason why I suggested the wait here. If we are still starting up, there's no point for upstream to call reconnect right?

Reply:
But we have no idea why `config.MDServer()` is `nil`. Also this is IMO the wrong place for a retry; if `mdServer` is `nil`, then we aren't reachable by definition.